pkgs/applications/graphics/structorizer/default.nix

```
@@ -0,0 +1,84 @@
+{ stdenv, lib, fetchFromGitHub, jdk11, makeDesktopItem, makeWrapper
+, copyDesktopItems }:
```

```suggestion
{ stdenv
, lib
, fetchFromGitHub
, jdk11
, makeDesktopItem
, makeWrapper
, copyDesktopItems
}:
```
If you have so many inputs that they're on multiple lines, you should put each on a new line IMO. Also makes it better to see what's going on when inputs change.